integration-cli/docker_cli_run_test.go

```diff
@@ -3191,3 +3191,18 @@ func (s *DockerSuite) TestRunPublishPort(c *check.C) {
 		c.Fatalf("run without --publish-all should not publish port, out should be nil, but got: %s", out)
 	}
 }
+
+// Issue #10184.
+func (s *DockerDaemonSuite) TestDevicePermissions(c *check.C) {
+	const permissions = "crw-rw-rw-"
+	if err := s.d.StartWithBusybox(); err != nil {
+		c.Fatalf("could not start daemon with busybox: %v", err)
+	}
+	out, err := s.d.Cmd("run", "--device", "/dev/fuse:/dev/fuse:mrw", "busybox:latest", "ls", "-l", "/dev/fuse")
```

Comment:
change this to `out, err := dockerCmd(c, "run", "--device", "/dev/fuse:/dev/fuse:mrw", "busybox", "ls", "-l", "/dev/fuse")`